package.json

```
@@ -6,6 +6,9 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.2.1",
+    "bootstrap": "^4.6.0",
+    "jquery": "^3.5.1",
```

すみません！Bootstrapの方でした！

```suggestion
    "bootstrap": "^4.5.1",
    "jquery": "^3.6.0",
```